experimental/PropertyDDS/packages/property-shared-tree-interop/src/schemaConverter.ts

```diff
@@ -294,11 +292,19 @@ export function convertPSetSchemaToSharedTreeLls(
 		} else {
 			const fieldKind =
 				splitTypeId.context === "array" ? FieldKinds.sequence : FieldKinds.optional;
-
```

Comment:
The following code block addition is correct, I only do not understand how it worked before (it generated correctly the array of inherited types even without this block.

Reply:
It didn't work before. As I said, the corresponding test was false-positive. It's very easy to reproduce: just comment out this block and run the test.

Reply:
I have some older generation as follows. I wonder how it could generate "types": **["Test:Row-1.0.0", "Test:ExtendedRow-1.0.0"].** Nevertheless I do not need to investigate anymore, just curious.
    "array<Test:Row-1.0.0>": {
      "localFields": {
        "": {
          "kind": "Sequence",
          "types": ["Test:Row-1.0.0", "Test:ExtendedRow-1.0.0"]
        }
      },
      "globalFields": [],
      "extraLocalFields": {"kind": "Forbidden", "types": []},
      "extraGlobalFields": false,
      "value": 0
    },

Reply:
Well, I see this line exists in your initial PR https://github.com/dstanesc/FluidFramework/blob/61cd9736afef11d564550bd16361f4530588f94b/experimental/PropertyDDS/packages/property-dds/src/sharedtree/llsSchemaConverter.ts#L330
but then it was forgotten in https://github.com/dstanesc/FluidFramework/pull/3/commits/4ccd97080504ee8c77104937eedc4fcb44c554e5